scripts/decode_credentials

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+gcloud kms decrypt \
```

Comment:
so... cloudbuild is able to decrypt these files. presumably it can do so because it sits inside our cloud organization?

if so, doesn't that let just about anybody get access to the raw credentials? all they need to do is add a console.log line to the test files, push to github, and cloudbuild will gladly decode and print out the credentials.

on the other hand, if we're fine with that, then do we need to encrypt them at all?

Reply:
As discussed:
- Only we can see the cloud build logs
- Only we can create branches inside the dataform repo and therefore decrypt the keys

If anyone else needs to run the integration tests, I guess we can create a branch for them inside dataform with their code